From: <ohering@suse.de>
Date: Fri, 27 Feb 2015 11:25:59 -0800
Patch-mainline: never, kabi
Subject: kabi: clockevents: export clockevents_unbind again
References: bnc#937888

In 3.12.65, commit 0d6a2bb674f96c9463ce7daefb9b5837adfb7194
(clockevents: export clockevents_unbind_device instead of
clockevents_unbind), upstream commit
32a158325acf12842764b1681f53903673f2f22e exported the correct public
function. Unfortunately the static variant made it into the kabi list.
Just reexport it.

--- a/kernel/time/clockevents.c
+++ b/kernel/time/clockevents.c
@@ -371,6 +371,7 @@
 	mutex_unlock(&clockevents_mutex);
 	return ret;
 }
+EXPORT_SYMBOL_GPL(clockevents_unbind);
 EXPORT_SYMBOL_GPL(clockevents_unbind_device);
 
 /**
